helpdesk_mgmt/controllers/main.py

```diff
@@ -57,6 +70,9 @@ def submit_ticket(self, **kw):
             .id,
             "partner_id": request.env.user.partner_id.id,
         }
+        if company.helpdesk_mgmt_new_ticket_teams:
+            team = http.request.env["helpdesk.ticket.team"].browse(int(kw.get("team")))
```

Comment:
This lets the user to set whatever team he please. Maybe we should check that it's a valid group.

Reply:
I understand what you mean, but I don't think it's necessary.
Let's say I want to create a ticket to the support team (to which I don't belong), I should be able to create it, don't you think?

Reply:
It isn't critical indeed but it's a good custom to be picky with user input :sweat_smile: In this case a user could create a ticket in an archived team (randomly picking the team id or knowing it before hand).

Reply:
What about this?

```suggestion
            team = http.request.env["helpdesk.ticket.team"].search([("id", "=", int(kw.get("team")), ("show_in_portal", "=", True)])
```